plugins/woocommerce-admin/client/marketing/hooks/useInstalledPlugins.ts

```diff
@@ -7,15 +7,16 @@ import { useSelect, useDispatch } from '@wordpress/data';
  * Internal dependencies
  */
 import { STORE_KEY } from '~/marketing/data/constants';
-import { Plugin } from './types';
+import { Plugin } from '~/marketing/types';
 
 export type UsePluginsType = {
 	installedPlugins: Plugin[];
 	activatingPlugins: string[];
 	activateInstalledPlugin: ( slug: string ) => void;
+	loadInstalledPluginsAfterActivation: ( slug: string ) => void;
 };
 
-export const usePlugins = (): UsePluginsType => {
+export const useInstalledPlugins = (): UsePluginsType => {
```

Comment:
Any particular reason for this name change?  It looks like this returns more than just installed plugins and the previous name may be more accurate.

Reply:
In this one I'm going to agree with @ecgan solution since for me it seems that the return is only related to the  installed plugins. usePlugins seems more generic for me and is not really clear.

💅  If we are going to refactor it tho... besides the name it would be even more helpful to add a JSDOC at the top :)

Reply:
I think in this case, the logic in here does not just deal with "installed" plugins.  It also deals with plugins that are in the process of activation, activating plugins, and "loading" plugins after installation.  While this semantically nitpicky, the semantics here are important.

I don't want to dig into this too much in this PR since I know it was not the intention of this PR to open this can of worms, but I don't think a unique data store should have been created around plugin activation specific to marketing.  We have a [data store for plugins](https://github.com/woocommerce/woocommerce/blob/2e1a73a6dc990ba101693a5802470f68e6f9b1c5/packages/js/data/src/plugins/index.ts) in WooCommerce that would work well here.  We should strive to not reinvent the wheel and consolidate efforts behind existing packages.

Reply:
> Any particular reason for this name change?

The main reason is because we already have a [`useRecommendedPlugins`](https://github.com/woocommerce/woocommerce/blob/0482860c74bffa60a11c5489b649b21796b253df/plugins/woocommerce-admin/client/marketing/overview-multichannel/DiscoverTools/useRecommendedPlugins.ts) for the "Discover more marketing tools" card. If we continue to have `usePlugin`, it feels confusing.

I renamed it to `useInstalledPlugins` because it is used in the "Installed extensions" card.

> I think in this case, the logic in here does not just deal with "installed" plugins. It also deals with plugins that are in the process of activation, activating plugins, and "loading" plugins after installation. While this semantically nitpicky, the semantics here are important.

Yeah, I thought about that too, but I couldn't think of a nice hook name for it. 

In the end, I named it `useInstalledPlugins` because it is mainly used in the "Installed extensions" card (although it is also used in the DiscoverTools component).